client/chainlink_models.go

```diff
@@ -389,9 +390,55 @@ type OCRTaskJobSpec struct {
 	ObservationSource        string        `toml:"observationSource"`                      // List of commands for the chainlink node
 }
 
+type P2PData struct {
+	RemoteIP string
+	PeerID   string
+}
+
 func (o *OCRTaskJobSpec) Type() string { return "offchainreporting" }
 func (o *OCRTaskJobSpec) String() (string, error) {
-	fluxMonitorTemplateString := `type = "offchainreporting"
+	// Pre-process P2P data for easier templating
```

Comment:
This feels dirty, but doing it withing the template felt dirtier, open to suggestions on this

Reply:
I like it, putting a list of clients within the model for it then to be parsed in the templates avoids duplicating this for loop throughout a bunch of tests.